src/app/pages/send/send-crypto-asset-form/components/amount-field.tsx

```diff
@@ -102,7 +102,7 @@ export function AmountField({ balance, bottomInputOverlay }: AmountFieldProps) {
           {meta.error}
         </ErrorLabel>
       )}
-      {bottomInputOverlay}
+      {bottomInputOverlay ?? <Box size="36px" />}
```

Comment:
Don't understand this change?

Reply:
If it didn't have the `Send all` button then the space in the UI collapsed and didn't look correct.